internal/factsengine/gatherers/ascsers_cluster_test.go

```diff
@@ -280,7 +280,7 @@ func (suite *AscsErsClusterTestSuite) TestAscsErsClusterGather() {
 
 	entries := suite.cache.Entries()
 	expectedEntries := []string{
-		"cibadmin",
+		"/usr/sbin/cibadmin",
```

Comment:
Should this be here?
The cache only needs to know that `cibadmin` output was stored, not the full path most probably

Reply:
That's a nasty typo, I'm glad you found it. Fixed in https://github.com/trento-project/agent/pull/421/commits/14909a1fcde6eb56f24e95d4298db38f2537f1b5